src/MSAL.PCL/Internal/Requests/BaseRequest.cs

```diff
@@ -41,13 +42,17 @@ internal abstract class BaseRequest
         internal readonly AuthenticationRequestParameters AuthenticationRequestParameters;
         internal readonly Authority Authority;
         internal readonly TokenCache TokenCache;
+        protected Exception Exception;
```

Comment:
This is an artifact from old ADAL flows. Great catch! I will remove it and update logic

---
In reply to: [93717304](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/138#discussion_r93717304) [](ancestors = 93717304)